bash-preexec.sh

```diff
@@ -38,10 +38,14 @@ if [[ -z "${BASH_VERSION:-}" ]]; then
 fi
 
 # Avoid duplicate inclusion
-if [[ "${__bp_imported:-}" == "defined" ]]; then
+if [[ "${bp_imported:-}" == "defined" ]]; then
     return 0
 fi
-__bp_imported="defined"
+bp_imported="defined"
```

Comment:
This is also our opportunity to use a clearer name, rather than just stripping the `__`. `bash_preexec_imported`, for instance. @rcaloras in case you have any naming preferences.

Reply:
👍 agree with suggested naming.